README.md

```diff
@@ -85,6 +85,20 @@ The provided code allows users to resize images to specified dimensions. This is
 * Just do `npm init -y` and `npm install express ws` on the client and server folders seperately and run the clients and server using `node index.js`.
 * They also asked to make the messages appear in a webpage. For this there is a ***index.html*** in the client folder which has html and the client JS code embedded inside it.
 
+
+### QR Code Reader (In Real Time)
+
+* This project demonstrates a simple Python-based QR code scanner that works in real-time using a webcam. It uses the `OpenCV` and `Pyzbar` libraries to detect and decode QR codes live from a video stream and save the results to a text file.
+* Real-time QR code detection using a webcam.
+* Decodes and saves QR code data to a text file.
+* Automatically shuts down the camera after successfully reading a QR code.
+* Uses the `OpenCV` library for video capture and the `Pyzbar` library for QR code decoding.
+* Supports various QR code formats.
+

```

Comment:
why this space but its ok. Also It looks the file is not synced to the latest readme. Its still showing the old layout but showing only the new lines as newly added so maybe I'll merge and check